src/common/addlayers/RegistryLayersDirective.js

```diff
@@ -269,6 +291,15 @@
 
             // load saved registry layers if they exist
             scope.addRegistryLayersFromSavedMap();
+
+            function addCartLayersInMapPreview(layerConfig) {
+              layerConfig.CRS = ['EPSG:4326'];
```

Comment:
Should this be hardcoded to EPSG:4326, shouldnt this be pulling this from the basemap layer?